### DISCUSSION TITLE
Datawriter QoS Transport_priority support under RTPS_UDP transport.

### DISCUSSION THREAD

**Discussion by z79523** [OpenDDS/OpenDDS#4457] [Q&A] [answered]
Hi Team,
I noticed that transport_priority was supported under UDP transport on latest version. And is there a plan to support datawriter qos transport_priority based on RTPS_UDP transport? If yes, when will or which version will support that?
Thanks!

**Comment by jrw972** [comment, marked as answer]
There is currently not a plan to support `transport_priority` for the `rtps_udp` transport.  We would certainly accept such a contribution and are interested in how users would leverage `transport_priority`.

**Reply by iguessthislldo** [reply]
Support for transport priority QoS only for udp and tcp (I think) transports has been noted in #4520.
